src/coreclr/tools/Common/TypeSystem/IL/Stubs/UnsafeIntrinsics.cs

```diff
@@ -119,6 +129,26 @@ private static MethodIL EmitAdd(MethodDesc method)
             return emit.Link(method);
         }
 
+        private static MethodIL EmitSubtract(MethodDesc method)
+        {
+            Debug.Assert(method.Signature.IsStatic && method.Signature.Length == 2);
+
+            TypeSystemContext context = method.Context;
+
+            ILEmitter emit = new ILEmitter();
+            ILCodeStream codeStream = emit.NewCodeStream();
+
+            codeStream.Emit(ILOpcode.ldarg_1);
+            codeStream.Emit(ILOpcode.sizeof_, emit.NewToken(context.GetSignatureVariable(0, method: true)));
+            codeStream.Emit(ILOpcode.conv_i);
+            codeStream.Emit(ILOpcode.mul);
+            codeStream.Emit(ILOpcode.ldarg_0);
+            codeStream.Emit(ILOpcode.sub);
```

Comment:
This one needs to be fixed too.

Reply:
Yup, another silly error, thanks for spotting it.